servermaster/jobmanager.go

```diff
@@ -57,29 +57,32 @@ func (jm *JobManagerImplV2) CancelJob(ctx context.Context, req *pb.CancelJobRequ
 func (jm *JobManagerImplV2) SubmitJob(ctx context.Context, req *pb.SubmitJobRequest) *pb.SubmitJobResponse {
 	log.L().Logger.Info("submit job", zap.String("config", string(req.Config)))
 	resp := &pb.SubmitJobResponse{}
-	var masterConfig *model.JobMaster
+	var (
+		id  lib.WorkerID
+		err error
+	)
+	// CreateWorker here is to create job master actually
+	// TODO: use correct worker cost
 	switch req.Tp {
 	case pb.JobType_Benchmark:
-		masterConfig = &model.JobMaster{
-			Tp:     model.Benchmark,
-			Config: req.Config,
+		config := &cvs.Config{}
+		err = json.Unmarshal(req.Config, config)
+		if err != nil {
+			break
 		}
+		id, err = jm.BaseMaster.CreateWorker(
+			lib.CvsJobMaster, config, defaultJobMasterCost)
 	default:
 		err := errors.ErrBuildJobFailed.GenWithStack("unknown job type", req.Tp)
 		resp.Err = errors.ToPBError(err)
 		return resp
 	}
-	// CreateWorker here is to create job master actually
-	// TODO: use correct worker type and worker cost
-	id, err := jm.BaseMaster.CreateWorker(
-		registry.WorkerTypeFakeMaster, masterConfig, defaultJobMasterCost)
```

Comment:
What about keeping the fake master, as an empty dry run task, without any dependency and easy for debugging.